data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
       "Янсен",
       "Янси",
       "Ян",
-      "Ярбро"
+      "Ярбро",
+      [ "Авель", "Abel" ],
+      [ "Авраам", "Abraham" ],
+      [ "Акерс", "Akers" ],
+      [ "Аланиз", "Alaniz" ],
+      [ "Александр", "Alexander" ],
+      [ "Алли", "Alley" ],
+      [ "Диамант", "Diamond" ],
+      [ "Амайя", "Amaya" ],
+      [ "Амарал", "Amaral" ],
+      [ "Амос", "Amos" ],
+      [ "Андрей", "Andrew" ],
+      [ "Арии", "Arias" ],
+      [ "Аруда", "Arruda" ],
```

Comment:
Probably not ideal that the second R got lost in translation. Names are supposed to be transcribed as close to the original as possible, at least in the modern tradition.